Dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG PYTHON_VERSION=3.8
+
+FROM python:${PYTHON_VERSION}
+
+ENV ENV="Local"
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt .
+
+RUN pip install -r requirements.txt
+
+COPY . .
+
+RUN python3 manage.py migrate
```

Comment:
Running `manage.py migrate` during the build process can cause issues if the database isn't available or properly configured at that time. Consider moving this command to the container startup process instead.